omod/src/main/webapp/resources/htmlforms/mexico/consult.xml

```diff
@@ -1046,6 +1046,24 @@
         </h1>
     </ifMode>
 
+    <script type="text/javascript">
+        // Hack to make "Confirmado" appear as "1a vez"
+        // See https://pihemr.atlassian.net/browse/MEX-459
+
+        $('body').on('DOMNodeInserted', 'div', function () {
+            const targetText = "1a vez";
+            const confirmedLabel = jq(jq('input[data-ng-model="d.confirmed"]')[0]).parent();
```

Comment:
@mogoodrich Good thought about the lookup by label. I've changed it to refer to a data property of the <input> tag it contains.